### PR TITLE
Feature/UI changes 01

### PR DESCRIPTION
**Pull request by rdunlocked18**
Hi @4shutosh Please review and Let me know your feedback.

Details: Update UI on almost all pages.

## Screenshots

WhatsApp Image 2021-05-12 at 6 34 43 PM
WhatsApp Image 2021-05-12 at 6 34 43 PM (1)
WhatsApp Image 2021-05-12 at 6 34 43 PM (2)
WhatsApp Image 2021-05-12 at 6 34 43 PM (3)
WhatsApp Image 2021-05-12 at 6 34 43 PM (4)

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -20,9 +19,14 @@
...
+By accessing this website we assume you accept these terms and conditions. Do not continue to use medscape if you do not agree to take all of the terms and conditions stated on this page.
+The following terminology applies to these Terms and Conditions, Privacy Statement and Disclaimer Notice and all Agreements: Client, You and Your refers to you, the person log on this website and compliant to the Company\'s terms and conditions. The Company, Ourselves, We, Our and Us, refers to our Company. , Parties, or Us, refers to both the Client and ourselves. All terms refer to the offer, acceptance and consideration of payment necessary to undertake the process of our assistance to the Client in the most appropriate manner for the express purpose of meeting the Client\'s needs in respect of provision of the Company\'s stated services, in accordance with and subject to, prevailing law of Netherlands. Any use of the above terminology or other words in the singular, plural, capitalization and/or he/she or they, are taken as interchangeable and therefore as referring to same.Cookies
```
Comment (4shutosh) [inline app/src/main/res/values/strings.xml:30]: Can we add this text to firebase?

**Review by 4shutosh — COMMENTED** [review]
Great work @rdunlocked18. 
Anything else that you would like to commit? Or is this final and completed?

**Comment by rdunlocked18** [comment]
> Great work @rdunlocked18. 
> Anything else that you would like to commit? Or is this final and completed?

Final commit

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (rdunlocked18) [reply app/src/main/res/values/strings.xml:30]: No need let it be local

**Pull request merged**